src/blosc2/lazyexpr.py

```diff
@@ -1904,6 +1977,30 @@ def lazyudf(func, inputs, dtype, chunked_eval=True, **kwargs):
     out: :ref:`LazyUDF`
         A :ref:`LazyUDF` is returned.
 
+    Examples
+    --------
+    >>> import blosc2
+    >>> import numpy as np
+    >>> # Define a user-defined function that will be applied to each block of data
+    >>> def my_function(inputs_tuple, output, offset):
+    >>>     a, b = inputs_tuple
```

Comment:
For the sake of locality (which improves readability), I'd move this function definition as close as possible to their use down below.